bool is_rigid_object
sensor_msgs/Image[] masks
float32[] masked_depth_std
---
MaskedObjectAttributes[] moas_list
